src/swapd/swap_state.rs

```diff
@@ -181,14 +179,17 @@ pub enum SwapStateMachine {
     /*
         Bob Happy Path States
     */
-    // BobReveal state - transitions to BobFunded on event AddressTransaction,
+    // BobReveal state - transitions to BobFeeEstimated on event FeeEstimation,
     // or Bob AwaitingBitcoinSweep on request AbortSwap or in case of incorrect
-    // funding amount.  Sends FundingCompleted to Farcasterd, Reveal to
+    // funding amount. Sends FundingCompleted to Farcasterd, Reveal to
     // counterparty peer, watches Lock, Cancel and Refund, checkpoints the Bob
     // pre Lock state, and sends the CoreArbitratingSetup to the counterparty
     // peer.
     #[display("Bob Reveal")]
     BobReveal(BobReveal),
+    // BobFeeEstimated state - transitions to BobFunded on event AddressTransaction.
```

Comment:
or Bob AwaitingBitcoinSweep on request AbortSwap or in case of incorrect funding amount.